<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>example_widg</class>
 <widget class="QWidget" name="example_widg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>582</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Message to Log:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="msg_input">
         <property name="text">
          <string>Hello World!</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="debug_button">
       <property name="text">
        <string>DEBUG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="info_button">
       <property name="text">
        <string>INFO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="warning_button">
       <property name="text">
        <string>WARNING</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="error_button">
       <property name="text">
        <string>ERROR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="critical_button">
       <property name="text">
        <string>CRITICAL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="mpl_widg" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plot_button">
       <property name="text">
        <string>Plot Random</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
